ogre/src/OgreScene.cc

```diff
@@ -502,6 +502,13 @@ MeshPtr OgreScene::CreateMeshImpl(unsigned int _id, const std::string &_name,
   return this->CreateMeshImpl(_id, _name, descriptor);
 }
 
+//////////////////////////////////////////////////
+MeshPtr OgreScene::CreateEllipsoidImpl(
+  unsigned int _id, const std::string &_name)
+{
+  return this->CreateMeshImpl(_id, _name, "unit_ellipsoid");
```

Comment:
As discussed in https://github.com/ignitionrobotics/ign-common/pull/165, this could just be `unit_sphere`